numpy/core/shape_base.py

```diff
@@ -183,23 +183,26 @@ def vstack(tup):
     """
     Stack arrays in sequence vertically (row wise).
 
-    Take a sequence of arrays and stack them vertically to make a single
-    array. Rebuild arrays divided by `vsplit`.
+    This is equivalent to concatenation along the first axis after 1d arrays of
+    shape `(N,)` have been reshaped to `(1,N)`. Rebuilds arrays divided by
+    `vsplit`.
 
-    This function continues to be supported for backward compatibility, but
-    you should prefer ``np.concatenate`` or ``np.stack``. The ``np.stack``
-    function was added in NumPy 1.10.
+    This function makes most sense for arrays with 3 dimensions or less. For
```

Comment:
Should be "fewer", not "less"

Reply:
No, "less" is correct and preferred in the "N items or less" construction.

http://languagelog.ldc.upenn.edu/nll/?p=2819

Reply:
The comments on the blog post seem pretty divided, but it's not worth arguing over here.

Having said that, I think "up to 3 dimensions" would read better than either of those. But this patch is good enough without that.